src/engine_core/src/input/KeyCode.hpp

```diff
@@ -141,22 +141,22 @@ enum class EKeyCode : KeyCode
 
     NUMLOCKCLEAR = 83, /**< num lock on PC, clear on Mac keyboards
                         */
-    KP_DIVIDE = 84,
-    KP_MULTIPLY = 85,
-    KP_MINUS = 86,
-    KP_PLUS = 87,
-    KP_ENTER = 88,
-    KP_1 = 89,
-    KP_2 = 90,
-    KP_3 = 91,
-    KP_4 = 92,
-    KP_5 = 93,
-    KP_6 = 94,
-    KP_7 = 95,
-    KP_8 = 96,
-    KP_9 = 97,
-    KP_0 = 98,
-    KP_PERIOD = 99,
+    KpDivide = 84,
+    KpMultiply = 85,
+    KpMinus = 86,
```

Comment:
Thanks for renaming these 🙏🏻

Reply:
I would like to keep it consistent, but clang-tidy kept the SCREAMING on most of them